proxy/src/http/project.rs

```diff
@@ -341,6 +394,30 @@ impl ToDocumentedType for CreateInput {
     }
 }
 
+/// Bundled input data for project checkout.
+#[derive(Deserialize, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct CheckoutInput {
+    /// Location on the filesystem where the working copy should be created.
+    path: PathBuf,
+    /// Which peer are we checking out from. If it's `None`, we're checking out our own project.
+    peer_id: Option<coco::PeerId>,
```

Comment:
Note: I'm actually wrong about this. The `LocalUrl` is just the `Hash` of the project. Picking which `HEAD` is another issue: https://github.com/radicle-dev/radicle-link/issues/234